Leetcode/EigenDemo/eigenPractice.hpp

```diff
@@ -0,0 +1,28 @@
+//
+//  eigenPractice.hpp
+//  EigenDemo
+//
+//  Created by dsh on 2021/7/18.
+//
+
+#ifndef eigenPractice_hpp
+#define eigenPractice_hpp
+
+#include <stdio.h>
+#include <iostream>
+
+#include "Eigen/core"
+#include "Eigen/Dense"
+
+using namespace Eigen;
+
+#define MATRIX_SIZE 50
+
+class eigenPractice {
+public:
+    eigenPractice() {};
+    void eigenBasic();
+    void eigenGeometry();
+};
+
+#endif /* eigenPractice_hpp */
```

Comment:
```suggestion
#endif  // eigenPractice_hpp
```

https://google.github.io/styleguide/cppguide.html#Namespace_Formatting

也许可以跟着 namespace 的规矩来。至少遵守行位注释两格空格的规则。